File: app/src/main/java/org/odddev/fantlab/core/di/PresentersModule.java
package org.odddev.fantlab.core.di;

import dagger.Module;
import dagger.Provides;
import rx.subscriptions.CompositeSubscription;

/**
 * @author kenrube
 * @date 16.09.16
 */

@Module
public class PresentersModule {

    @Provides
    CompositeSubscription provideCompositeSubscription() {
        return new CompositeSubscription();
    }
}

File: app/src/main/java/org/odddev/fantlab/core/di/PresentersModule.kt
package org.odddev.fantlab.core.di

import dagger.Module
import dagger.Provides
import rx.subscriptions.CompositeSubscription

/**
 * @author kenrube
 * *
 * @since 16.09.16
 */

@Module
class PresentersModule {

	@Provides
	fun provideCompositeSubscription(): CompositeSubscription = CompositeSubscription()
}
